contrib/babelfishpg_tsql/sql/fts_contains_pgconfig.sql

```diff
@@ -1,12 +1,20 @@
 -- Given the query string, determine the Postgres full text configuration to use
--- Currently we only support simple terms, so the function simply returns 'fts_contains'
+-- Currently we only support simple terms and prefix terms
+-- For simple terms, we use the 'fts_contains' configuration
+-- For prefix terms, we use the 'simple' configuration
+-- They are the configurations that provide closest matching according to our experiments
 CREATE OR REPLACE FUNCTION sys.babelfish_fts_contains_pgconfig(IN phrase text)
   RETURNS regconfig AS
 $$
 DECLARE
   joined_text text;
   word text;
 BEGIN
+  -- Prefix term
+  IF (phrase COLLATE C) SIMILAR TO ('[ ]*"%\*"[ ]*' COLLATE C) THEN
```

Comment:
SELECT * FROM t WHERE CONTAINS(txt, '"blu*"')
is equivalent to
SELECT * FROM t WHERE CONTAINS(txt, '     "blu*"   ') in SQL Server, presumably because the string constant is parsed by a grammar internally. This means that we can allow spaces in the search term. 
'[ ]*"%\*"[ ]*' says that the search term can has any number of preceding or trailing white spaces. And it should have something surrounded by a pair of double quote. Inside the double quote, it should be any string followed by a * character.